numpy/ma/core.py

```diff
@@ -4240,22 +4240,33 @@ def get_real(self):
         return result
     real = property(fget=get_real, doc="Real part")
 
-    def count(self, axis=None):
+    def count(self, axis=None, keepdims=np._NoValue):
         """
         Count the non-masked elements of the array along the given axis.
 
         Parameters
         ----------
-        axis : int, optional
-            Axis along which to count the non-masked elements. If `axis` is
-            `None`, all non-masked elements are counted.
+        axis : None or int or tuple of ints, optional
+            Axis or axes along which the count is performed.
+            The default (`axis` = `None`) is perform the count sum over all
```

Comment:
Are single ticks OK here (honestly not sure)? The sentence reads a bit funny. "is to perform" or "performs" sound fine to me, though who knows english has some weird grammar ;).